src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,61 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final int NO_CARS_IN_GAME = 0;
+    private static final int NO_DUPLICATE = 0;
+
+    private ArrayList<Car> carsInGame = new ArrayList<Car>();
```

Comment:
```suggestion
    private final List<Car> carsInGame = new ArrayList<>();
```
제네릭표기에서 생략가능하고, 구체타입보다는 인터페이스로 참조해보는건 어떨까요? 
아래부분들도 마찬가지입니다!